cmd/setup/steps.yaml

```diff
@@ -1,41 +1,42 @@
 FirstInstance:
-  MachineKeyPath:
-  PatPath:
-  InstanceName: ZITADEL
-  DefaultLanguage: en
+  MachineKeyPath: # ZITADEL_FIRSTINSTANCE_MACHINEKEYPATH
+  PatPath: # ZITADEL_FIRSTINSTANCE_PATPATH
```

Comment:
Unfortunately these variables are here and not directly as "path" under the config of the machine account itself. Would it make sense to explain that these variables have to be set in combination with vars below?

Reply:
done